packages/meta-updater/src/updateLanguageScopeSupportConfig.ts

```diff
@@ -0,0 +1,25 @@
+import type { FormatPluginFnOptions } from "@pnpm/meta-updater";
+
+export function updateLanguageMdxConfig(
+  languageId: string,
+  actual: string | null,
+  options: FormatPluginFnOptions,
+): string | null {
+  if (options.manifest.name !== "@cursorless/cursorless-org-docs") {
+    return null;
+  }
+
+  if (actual != null) {
+    return actual;
+  }
```

Comment:
huh interesting. I guess this means we're just auto-generating at the start but not trying to clobber if they change. Seems ok. Will be annoying if we change the `<Language>` element, but useful to be able to be a bit more custom with languages I guess

Reply:
Exactly that was my intent. If we don't want anything custom per language then we don't really need mdx at all surfing this is the right direction.

Reply:
Yeah seems reasonable